tests/test_nodedev.py

```diff
@@ -155,6 +155,15 @@ def testPCIMdev():
     assert dev.device_type == "mdev"
     assert dev.type_id == "nvidia-11"
 
+def testPCIMdevNewFormat():
+    conn = utils.URIs.open_testdriver_cached()
+    devname = "mdev_35ceae7f_eea5_4f28_b7f3_7b12a3e62d3c_0000_06_00_0"
+    dev = _nodeDevFromName(conn, devname)
+    assert dev.name == devname
+    assert dev.parent == "pci_0000_06_00_0"
+    assert dev.device_type == "mdev"
+    assert dev.type_id == "nvidia-11"
+
```

Comment:
So, I just realized that this additional test does not really test what I was trying to test. If I add the following line to the end of the test
```
    assert dev.get_mdev_uuid() == "35ceae7f-eea5-4f28-b7f3-7b12a3e62d3c"
```

then the test will fail if the test is run against libvirt versions earlier than 7.3.0. This is because the `<uuid>` field will not be formatted in the xml description of `dev`, so `dev.get_mdev_uuid()` will return `None`.  I'm not sure about the best way to test this change...

Reply:
Can you just set the test to be skipped on older libvirt versions in some way  ?